tests/conftest.py

```diff
@@ -41,6 +54,19 @@ def notebook_service(docker_ip, docker_services):
     return url, token
 
 
+@pytest.fixture(scope="session", autouse=True)
+def install_package(docker_compose):
+    # assurance for host user UID other that 1000
+    command = build_command(
+        "chown -R jovyan:users /home/jovyan/apps/aiidalab-widgets-base", user="root"
+    )
+    docker_compose.execute(command)
+
+    # install the package
+    command = build_command("pip install -U .")
```

Comment:
Do not use the `-U` upgrade option, that's not what `aiidalab install` does.

Reply:
I think here we are not going to mimic what the `aiidalab install` is doing. For example when we bump the version of a dependency we want to see the effect of that.